phylogenetic/rules/export.smk

```diff
@@ -17,6 +17,23 @@ This part of the workflow usually includes the following steps:
 See Augur's usage docs for these commands for more details.
 """
 
+rule prepare_auspice_config:
+    input:
+        auspice_template="config/auspice_config_template.json",
+    output:
+        auspice_config="results/config/auspice_config_{serotype}_{gene}.json",
+    params:
+        replace_clade_key=lambda wildcard: "clade_membership" if wildcard.gene in ['genome'] else "nextclade_subtype",
+        replace_clade_title=lambda wildcard: "Serotype" if wildcard.serotype in ['all'] else "DENV genotype",
+    shell:
+        """
+        cat {input.auspice_template} \
+        | sed "s/REPLACE_GENE/{wildcards.gene}/g" \
+        | sed "s/REPLACE_CLADE_KEY/{params.replace_clade_key}/g" \
+        | sed "s/REPLACE_CLADE_TITLE/{params.replace_clade_title}/g" \
```

Comment:
If you want to use `sed` you should probably ensure the replacement strings are properly escaped.